src/App.tsx

```diff
@@ -133,9 +133,27 @@ const TMP_DATA: MarkerFileData = {
 	}]
 };
 
-export default function App() {
+    export default function App() {
+	const [openFileSelector, {filesContent, loading}] = useFilePicker({
+		accept: '.txt',
+	});
+
+	if (loading) {
+		return <div>Loading...</div>;
+	}
+
 	return <div>
 		<h1>Hello, world!</h1>
+		<button onClick={() => openFileSelector()}>Select files</button>
+		<br />
+		{filesContent.map((file, index) => (
+			<div>
+				<h2>{file.name}</h2>
+				<div  key={index}>{file.content}</div>
```

Comment:
indices should basically never be used as keys, and this isn't the top-level of the map return anyway

Reply:
this belonged to a temporary section and has since been removed